library/forms.inc

```diff
@@ -1,16 +1,10 @@
 <?php
 $GLOBALS['form_exit_url'] = "javascript:parent.closeTab(window.name, false)";
 
-function getFormById($id, $cols = "*")
-{
-    $sql = "SELECT $cols FROM forms WHERE id = ? AND deleted = 0 ORDER BY date DESC limit 0,1";
-    //echo $sql . "<br />";
-    return sqlQuery($sql, array($id));
-}
 
 function getFormInfoById($id, $cols = "*")
```

Comment:
we can remove this function. the only use of this function is actually commented out